stl/inc/xutility

```diff
@@ -5971,8 +5971,12 @@ _NODISCARD _CONSTEXPR_BIT_CAST _Ty _Float_copysign(const _Ty _Magnitude, const _
 // FUNCTION TEMPLATE _Is_nan
 template <class _Ty, enable_if_t<is_floating_point_v<_Ty>, int> = 0>
 _NODISCARD _CONSTEXPR_BIT_CAST bool _Is_nan(const _Ty _Xx) { // constexpr isnan()
+#if defined(__LDBL_DIG__) && __LDBL_DIG__ == 18
+    return _CSTD _LDtest(&_Xx) == _NANCODE;

```

Comment:
This seems to be ill-formed when `_Ty` is `float` or `double`. (Also occurs on line 6005.)

Reply:
> This seems to be ill-formed when `_Ty` is `float` or `double`. (Also occurs on line 6005.)

Agreed.